ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_linalg.py

```diff
@@ -896,48 +896,41 @@ def test_jax_cond(
 
 # multi_dot
 @handle_frontend_test(
-    fn_tree="jax.lax.linalg.multi_dot",
+    fn_tree="jax.numpy.linalg.multi_dot",
     dtype_and_x=helpers.dtype_and_values(
         available_dtypes=helpers.get_dtypes("float"),
-        min_value=0,
-        max_value=10,
-        shape=helpers.ints(min_value=2, max_value=5).map(lambda x: tuple([x, x])),
+        num_arrays=3,
+        min_value=2,
+        max_value=5,
+        shape=helpers.ints(min_value=2, max_value=3).map(lambda x: tuple([x, x])),
     ).filter(
         lambda x: "float16" not in x[0]
         and "bfloat16" not in x[0]
+        and "bfloat64" not in x[0]
+        and "float64" not in x[0]
         and np.linalg.cond(x[1][0]) < 1 / sys.float_info.epsilon
         and np.linalg.det(np.asarray(x[1][0])) != 0
     ),
     test_with_out=st.just(False),
 )
-def test_jax_lax_multi_dot(
+def test_jax_multi_dot(
     *,
     dtype_and_x,
     on_device,
     fn_tree,
     frontend,
     test_flags,
+    backend_fw,
 ):
     dtype, x = dtype_and_x
-    x = np.asarray(x[0], dtype=dtype[0])
-
-    ret, frontend_ret = helpers.test_frontend_function(
+    x = np.asarray(x)
+    helpers.test_frontend_function(
         input_dtypes=dtype,
         frontend=frontend,
+        backend_to_test=backend_fw,
         test_flags=test_flags,
         fn_tree=fn_tree,
         on_device=on_device,
         test_values=False,
-        args=(x,),
-    )
```

Comment:
It's not clear to me why this test needs to be modified. If there's a problem with this test then would you please open a separate PR for this? It's not best practice to fix problems with other functions in one PR.